app/models/TulsaBlotter/offense.rb

```diff
@@ -0,0 +1,4 @@
+class TulsaBlotter::Offense < ApplicationRecord
+  belongs_to :arrest
+  validates :case_number, :court_date, :bond_type, :disposition, presence: true
```

Comment:
Pretty sure all of these fields will sometimes be null when a person is arrested. I think the `description` is the only one that should always be present.